nmrfx-analyst-gui/src/main/java/org/nmrfx/processor/gui/GUIScripter.java

```diff
@@ -71,7 +71,8 @@ PolyChart getChart() {
     }
 
     public String active() {
-        PolyChart chart = useChart != null ? useChart : PolyChart.getActiveChart();
+        PolyChart chart;
+        chart = useChart != null ? useChart : PolyChartManager.getInstance().getActiveChart();
```

Comment:
whats the purpose of this change? just formatting to not make the line as long?

Reply:
This wasn't done on purpose. 

This is likely a side effect of using intellij's "inline method" refactoring tool. I'll change it back manually before merging.